mask/eolearn/mask/masking.py

```diff
@@ -45,8 +55,8 @@ def __init__(self, input_features, output_feature, join_operation: Union[str, Ca
 
     def zip_method(self, *masks: np.ndarray) -> np.ndarray:
         """Joins masks using the provided operation"""
-        final_mask, *masks = masks
-        for mask in masks:
+        final_mask, *tmp = masks
```

Comment:
`tmp` is a very poor name. Consider `masks_to_join`

Reply:
agreed, updeted